app/javascript/components/pages/users/UsersConstants.jsx

```diff
@@ -1 +1,2 @@
-export const userTypes = ["Admin", "Member", "Guest"];
\ No newline at end of file
+export const userTypes = ["Admin", "Member", "Guest"];
+export const emailTypes = ["Display Email", "No Display Email"];

```

Comment:
```suggestion
export const emailTypes = ["Display Email", "Don't Display Email"];

```

Reply:
i will do do not display email, don't with apostrophe raises some issues

Reply:
wait do `"Dont\'t"` then